### DISCUSSION TITLE
DDOS harmful code at GIthub

### DISCUSSION THREAD

**Discussion by blagodatov** [community/community#64591] [Code Security] [answered]
### Select Topic Area

Bug

### Body

Urgent!
https://github.com/db1000n-coordinators/LoadTestConfig/
this is a harmful code for DDOS attacks.
Please block it ASAP.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @blagodatov 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
